.github/workflows/codeql-analysis.yml

```diff
@@ -5,7 +5,10 @@ on:
     branches: [ main ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ main ]
+    branches: [ main, feature/648-extend-codeql-scope ]
```

Comment:
**feature/648-extend-codeql-scope** added to trigger the CodeQL scan, will be removed before merging the PR